pkg/handler/handler_test.go

```diff
@@ -116,6 +115,84 @@ func TestRegisterOpenAPIVersionedService(t *testing.T) {
 	}
 }
 
+var updatedSwagger = []byte(`{
+  "swagger": "2.0",
+  "info": {
+   "title": "Kubernetes",
+   "version": "v1.12.0"
+  }}`)
+
+func getBodyOrDie(server *httptest.Server) string {
+	req, err := http.NewRequest("GET", server.URL+"/openapi/v2", nil)
+	if err != nil {
+		panic(fmt.Errorf("Unexpected error in creating new request: %v", err))
+	}
+
+	req.Header.Add("Accept", "application/json")
+	resp, err := server.Client().Do(req)
+	if err != nil {
+		panic(fmt.Errorf("Unexpected error in serving HTTP request: %v", err))
+	}
+
+	defer resp.Body.Close()
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		panic(fmt.Errorf("Unexpected error in reading response body: %v", err))
+	}
+	return string(body)
+}
+
+func normalizeSwaggerOrDie(j []byte) []byte {
```

Comment:
Thanks for refactoring this out. Can this be reused in the test above too? L26-35 I think